src/_guides/language/analysis-options.md

```diff
@@ -381,6 +381,44 @@ linter:
   You can, however, use the other syntax for rules in an included file.
 {{site.alert.end}}
 
+## Enabling analyzer plugins (experimental)
+
+The Analyzer package offers an experimental feature called Analyzer plugins.
+These plugins allow you to add linter rules and quick fixes that third-party
+plugins specify.
+You can use no more than one Analyzer plugin in each
+`analysis_options.yaml` file.
+Each analyzer plugin that you use increases how much memory the analyzer uses.
+
+Don’t use analyzer plugins if your situation meets either of the following
+conditions:
+
+* Your development machine has less than 16 GB of memory
+* You use a mono-repo with more than 10 `pubspec.yaml` and
+  `analysis_options.yaml` files.
+  
+You can find a few analyzer plugins on [pub.dev]({{site.pub}}).
```

Comment:
I haven't taken a look at the rest of this PR yet, but I wanted to ask if it would make sense to link to packages that depend on `package:analyzer_plugin` here?

https://pub.dev/packages?q=dependency%3Aanalyzer_plugin

Linking to just pub.dev may be confusing as they won't know how to find analyzer plugins from there.

Reply:
@parlough : I agree with that. Let me do that.